import "goby/moos/protobuf/goby_moos_app.proto";
import "goby/common/protobuf/option_extensions.proto";


message CTDSimpleConfig
{
    optional GobyMOOSAppConfig common = 1 [(goby.field).description="Configuration common to all Goby MOOS applications"]; // see goby_moos_app.proto

    required string serial_port = 2;
    optional int32 serial_baud = 3 [default = 9600];
    
}